src/ui/components/search/spellcheckcomponent.js

```diff
@@ -33,11 +33,6 @@ export default class SpellCheckComponent extends Component {
     return 'search/spellcheck';
   }
 
-  onCreate () {
-    this.core.persistentStorage.delete(StorageKeys.SKIP_SPELL_CHECK, true);
```

Comment:
Does this mean these parameters will stay in the URL when you click on the spellcheck?

Reply:
prior to this we were deleting them from globalStorage after the search in core.js and deleting them from persistenStorage on page load with the lines you commented on
now they're both deleted after a search. This is in the commit msg too! (but don't blame you for not missing since it's really long...)